src/content/docs/configuration/key-binding.mdx

```diff
@@ -49,6 +49,10 @@ eval "$(atuin init zsh)"
 
 You can then choose to bind Atuin if needed, do this after the call to init.
 
+## Enter key behavior
+
+By default, the `enter` key will directly execute the selected comamnd instead of letting you edit it like the `tab` key. If you want to change this behavior, set `enter_accept = true` in your config. For more details: [enter_accept](https://docs.atuin.sh/configuration/config/#enter_accept).
```

Comment:
```suggestion
By default, the `enter` key will directly execute the selected comamnd instead of letting you edit it like the `tab` key. If you want to change this behavior, set `enter_accept = false` in your config. For more details: [enter_accept](https://docs.atuin.sh/configuration/config/#enter_accept).
```